CHANGES

```diff
@@ -6,10 +6,14 @@
   select a netmod in order to build ch4, else configure will prompt
   the user to choose.
 
-# Further improvements to ch4 business card exchange: extra
-  long address support and fixes for PMIx integration.
+# Add support for Yaksa datatype engine.
 
-# Un-inline non-critical ch4 code for improved build times.
+# Add support for GPU buffers in pt2pt communication.
```

Comment:
This GPU bullet is probably underselling what has been added. Open to suggestion to improve it, or any other addition here.